CONTRIBUTING.md

```diff
@@ -187,7 +187,7 @@ function $DONE(arg) {
 
 ## Handling Errors and Negative Test Cases
 
-Expectations for **parsing errors** should be declared using [the `negative` frontmatter flag](#negative):
+Expectations for **parsing errors** should be declared using [the `negative` frontmatter flag](#negative). They must also include the exact `throw` statement specified in this example (in order to guarantee that implementations do not execute the code):
```

Comment:
I think it's fine. What I think we should improve eventually a way for contributors to use the linter before the CI running over the PR. Something that puts it more visible or clear.